scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -224,7 +228,30 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
                                             export_layout=export_layout,
                                             archive_output_format=archive_output_format,
                                             logger=logger)
-
+        if build_config == 'profile':
+            generate_launcher_startup_script(selected_platform, ctx.project_name, export_layout.launcher_type, export_layout.launch_flags, export_layout.output_path)
+
+def generate_launcher_startup_script(platform, project_name, launcher_type, launch_flags, output_path):
+    launcher_name = 'GameLauncher' if launcher_type == exp.LauncherType.GAME else \
+                    ('ServerLauncher' if launcher_type == exp.LauncherType.SERVER else 'UnifiedLauncher')
+    slash = '.\\' if platform == 'pc' else './'
+
+    # We do not use pathlib in this specific instance b/c its string conversion removes the initial './', which can trip up the shell script
+    exec_path = f"{slash}{project_name}.{launcher_name}{'.exe' if platform == 'pc' else ''}"
```

Comment:
isn't platform the asset platform? The script should be running based on current platform.  You can't build/export a linux project on a windows host, or vice-versa, so its should not rely on the asset platform. For android, you don't launch from a script, so that would be meaningless there anyways. This should be `os.sep` instead.
```suggestion
    # We do not use pathlib in this specific instance b/c its string conversion removes the initial './', which can trip up the shell script
    exec_path = f".{os.sep}{project_name}.{launcher_name}{'.exe' if platform == 'pc' else ''}"
```